tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -1164,8 +1164,12 @@ Status IrEmitterUnnested::EmitCublasLtMatmulThunkF8(mlir::Operation* op) {
                       GetAllocationSlice(matmul.getA()));
   TF_ASSIGN_OR_RETURN(BufferAllocation::Slice b,
                       GetAllocationSlice(matmul.getB()));
-  TF_ASSIGN_OR_RETURN(BufferAllocation::Slice c,
-                      GetAllocationSlice(matmul.getC()));
+  BufferAllocation::Slice c;
+
+  if (matmul.getBetaAttr().getValueAsDouble() != 0.0) {
+    TF_ASSIGN_OR_RETURN(c, GetAllocationSlice(matmul.getC()));
+  }
+
```

Comment:
The non-FP8 case doesn't have this logic. It unconditionally gets C, which IIUC is simply the output if there is no matrix bias. For consistency with the non-FP8 case, can we also uncondtionally get C here? This would also simply some logic in cublas_lt_matmul_think.cc and matmul_utils.h.